mobify/pikabu
# Fix/custom classes

Status: **Ready for Review**
Owner: @ericawright
Reviewers: @mikenikles @fractaltheory @jansepar @marlowpayne @stewartyu
## Changes
- user was unable to add custom classes for appendTo and container options (unless they also used pikabu's built in classes)
- $viewport was not being defined when custom classes used
- $animators array was missing custom class when .pikabu_container class was not there
## Todos:
### Feedback:

_none so far_
## How to Test
- in `pikabu/examples/assets/js/main`, add some custom options such as:
  `container: '.m-pikabu-container',
          appendTo: '.m-pikabu-viewport',`
  to the pikabu declarations (make sure to do this to all 4, or else it will look for the default elements which we will remove)
- in `pikabu/examples/index.html` change line 18 `<div class="pikabu">` remove "pikabu" and add custom class name: ".m-pikabu-viewport"
- also in `pikabu/examples/index.html` change line 25 `<div class="pikabu__container">` remove "pikabu_container" and replace with custom class "m-pikabu-container"
- run `grunt serve`
- go to http://localhost:3000/examples/
